packages/rest/src/providers/parse-params.provider.ts

```diff
@@ -36,3 +37,26 @@ export class ParseParamsProvider implements Provider<ParseParams> {
       });
   }
 }
+
+@bind(
+  asMiddleware({
+    group: 'parseParams',
+    upstreamGroups: 'findRoute',
+    chain: RestTags.REST_MIDDLEWARE_CHAIN,
```

Comment:
`parseParams`, `findRoute ` and other strings are being repeated, I think we should be using constants.